jdaviz/app.py

```diff
@@ -1244,11 +1231,29 @@ def _on_data_deleted(self, msg):
                 self.state.data_items.remove(data_item)
 
     @staticmethod
-    def _create_data_item(label):
+    def _create_data_item(data):
+        ndims = len(data.shape)
+        wcsaxes = data.meta.get('WCSAXES', data.meta.get('header', {}).get('WCSAXES', None))
```

Comment:
You can simplify this now that #1325 is merged. 🤞

Reply:
Thanks for the reminder!  I'll pull in your suggestion when I rebase.